rewriter/Minitest.cc

```diff
@@ -182,19 +182,30 @@ ast::ExpressionPtr getIteratee(ast::ExpressionPtr &exp) {
     }
 }
 
+ast::ExpressionPtr prepareTestEachBody(core::MutableContext ctx, core::NameRef eachName, ast::ExpressionPtr body,
+                                       ast::MethodDef::ARGS_store &args, ast::InsSeq::STATS_store destructuringStmts,
+                                       ast::ExpressionPtr &iteratee);
+
 // this applies to each statement contained within a `test_each`: if it's an `it`-block, then convert it appropriately,
 // otherwise flag an error about it
 ast::ExpressionPtr runUnderEach(core::MutableContext ctx, core::NameRef eachName,
-                                const ast::InsSeq::STATS_store &destructuringStmts, ast::ExpressionPtr stmt,
+                                ast::InsSeq::STATS_store &destructuringStmts, ast::ExpressionPtr stmt,
                                 ast::MethodDef::ARGS_store &args, ast::ExpressionPtr &iteratee) {
     // this statement must be a send
     if (auto *send = ast::cast_tree<ast::Send>(stmt)) {
         // the send must be a call to `it` with a single argument (the test name) and a block with no arguments
-        if (send->fun == core::Names::it() && send->numPosArgs() == 1 && send->hasBlock() &&
-            send->block()->args.size() == 0) {
-            // we use this for the name of our test
-            auto argString = to_s(ctx, send->getPosArg(0));
-            auto name = ctx.state.enterNameUTF8("<it '" + argString + "'>");
+        if ((send->fun == core::Names::it() && send->numPosArgs() == 1 && send->hasBlock() &&
+             send->block()->args.size() == 0) ||
+            ((send->fun == core::Names::before() || send->fun == core::Names::before()) && send->numPosArgs() == 0 &&
```

Comment:
You are checking for the same name here?  Was one of these supposed to be `beforeAngles`?